src/dialog-editor/services/modal/modalService.ts

```diff
@@ -81,6 +85,40 @@ class ModalController {
             }
           );
         }
+        // set modal title
+        if (!this.modalData.dynamic) {
+          var titleLabel;
+
+          switch (this.modalData.type) {
+            case 'DialogFieldTextBox':
+              titleLabel = "Text Box";
+              break;
+            case 'DialogFieldTextAreaBox':
+              titleLabel = "Text Area";
+              break;
+            case 'DialogFieldCheckBox':
+              titleLabel = "Check Box";
+              break;
+            case 'DialogFieldDropDownList':
+              titleLabel = "Dropdown";
+              break;
+            case 'DialogFieldRadioButton':
+              titleLabel = "Radio Button";
+              break;
+            case 'DialogFieldDateControl':
+              titleLabel = "Datepicker";
+              break;
+            case 'DialogFieldDateTimeControl':
+              titleLabel = "Timepicker";
+              break;
+            case 'DialogFieldTagControl':
+              titleLabel = "Tag Control";
+              break;
+          }
+          this.modalTitle =  __("Edit " + titleLabel + " Field");
```

Comment:
broken i18n .. you can *never* use + inside of `__`

Reply:
Ok, fixed the i18n issues.  -thanks